aeron-client/src/main/java/io/aeron/logbuffer/LogBufferDescriptor.java

```diff
@@ -217,7 +300,78 @@ public class LogBufferDescriptor
         offset += CACHE_LINE_LENGTH;
         LOG_DEFAULT_FRAME_HEADER_OFFSET = offset;
 
-        LOG_META_DATA_LENGTH = align(offset + LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH, PAGE_MIN_SIZE);
+        offset += LOG_DEFAULT_FRAME_HEADER_MAX_LENGTH ;
+
+        LOG_TERM_OFFSET_OFFSET = offset;
+        offset += SIZE_OF_INT;
+
+        LOG_IS_SPARSE_OFFSET = offset;
+        offset += SIZE_OF_BYTE;
+
+        LOG_IS_TETHER_OFFSET = offset;
+        offset += SIZE_OF_BYTE;
+
+        LOG_IS_REJOIN_OFFSET = offset;
+        offset += SIZE_OF_BYTE;
+
+        LOG_IS_RELIABLE_OFFSET = offset;
+        offset += SIZE_OF_BYTE;
+
+        LOG_SIGNAL_EOS_OFFSET = offset;
+        offset += SIZE_OF_BYTE;
+
+        // padding to ensure 4 byte aligned.
+        offset += 3;
+
+        // todo: will be removed
+        if (offset % 4 != 0)
+        {
+            throw new Error("Bad alignment: offset=" + offset);
+        }
+
+        LOG_SOCKET_RCVBUF_LENGTH_OFFSET = offset;
```

Comment:
Let's try to use available space after the `page_size` field (`LOG_PAGE_SIZE_OFFSET + SIZE_OF_INT`) where we still have 36 bytes before the default header starts.
That should be enough to fit 6 ints or 5 ints + 6 booleans.

Reply:
Done